docs/pbc/all/multi-factor-auth/202505.0/howto-create-custom-mfa.md

```diff
@@ -0,0 +1,204 @@
+     *
+     * @return bool
+     */
+    public function isApplicable(MultiFactorAuthTransfer $multiFactorAuthTransfer): bool
+    {
+        return $multiFactorAuthTransfer->getType() === static::YOUR_MFA_TYPE;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\MultiFactorAuthTransfer $multiFactorAuthTransfer
+     *
+     * @return \Generated\Shared\Transfer\MultiFactorAuthTransfer
+     */
+    public function send(MultiFactorAuthTransfer $multiFactorAuthTransfer): MultiFactorAuthTransfer
+    {
+        // Implement your code sending logic here
+        // For example, send via SMS, authenticator app, etc.
+        
+        return $multiFactorAuthTransfer;
+    }
+}
+```
+</details>
+
+## 3. Wire strategy in factory
+
+Add your strategy to the factory so that it can be used by the Multi-Factor Authentication system to resolve the correct sender strategy based on the type of MFA method selected by the customer.
+
+<details>
+<summary>Pyz\Zed\MultiFactorAuth\Business\MultiFactorAuthBusinessFactory.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Zed\MultiFactorAuth\Business;
+
+use Spryker\Zed\MultiFactorAuth\Business\MultiFactorAuthBusinessFactory as SprykerMultiFactorAuthBusinessFactory;
+use Pyz\Zed\MultiFactorAuth\Business\Strategy\Customer\YourMfaCodeSenderStrategy;
+
+class MultiFactorAuthBusinessFactory extends SprykerMultiFactorAuthBusinessFactory
+{
+    /**
+     * @return array<\Spryker\Zed\MultiFactorAuth\Business\Strategy\SendStrategyInterface>
+     */
+    protected function getCustomerCodeSenderStrategies(): array
+    {
+        return [
+            // ... other strategies
+            new YourMfaCodeSenderStrategy(),
+        ];
+    }
+}
+```
+</details>
+
+## 4. Register your plugins
+
+Register your plugin in the dependency provider:
+
+<details>
+<summary>Pyz\Yves\MultiFactorAuth\MultiFactorAuthDependencyProvider.php</summary>
+
+```php
+<?php
+
+namespace Pyz\Yves\MultiFactorAuth;
+
+use Spryker\Yves\MultiFactorAuth\MultiFactorAuthDependencyProvider as SprykerMultiFactorAuthDependencyProvider;
+
+class MultiFactorAuthDependencyProvider extends SprykerMultiFactorAuthDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\MultiFactorAuthExtension\Dependency\Plugin\MultiFactorAuthTypePluginInterface>
+     */
+    protected function getCustomerMultiFactorAuthPlugins(): array
+    {
+        return [
+            // ... other plugins
+            new YourMfaTypePlugin(),
+        ];
+    }
+}
+```
+</details>
+
+## Related Documentation
+
+- [Multi-Factor Authentication Feature Overview](/docs/pbc/all/multi-factor-auth/{{site.version}}/multi-factor-auth.html)
+- [Install the Multi-Factor Authentication Feature](/docs/pbc/all/multi-factor-auth/{{site.version}}/install-and-upgrade/install-multi-factor-auth.html)
+- [How to Install Customer Email Multi-Factor Authentication Method](/docs/pbc/all/multi-factor-auth/{{site.version}}/howto-install-customer-email-mfa.html)
```

Comment:
Add this to the related section in the front matter so it's rendered properly. Syntax example: https://github.com/spryker/spryker-docs/blob/3285b8340ef288380ba922f653a06736681c7b31/docs/dg/dev/architecture/conceptual-overview.md?plain=1#L10

Reply:
Done